force-app/main/default/classes/AwsUploader.cls

```diff
@@ -139,13 +143,15 @@ global class AwsUploader implements Database.Batchable<SObject>, Database.Allows
 
     @AuraEnabled
     public static void emailUser(String message) {
-        User currentUser = [SELECT Id, Email FROM User WHERE Id = :UserInfo.getUserId() LIMIT 1];
+        if(Schema.sObjectType.User.fields.Email.isAccessible()){
+        User currentUser = [SELECT Id, Email FROM User WHERE Id = :UserInfo.getUserId() WITH SECURITY_ENFORCED LIMIT 1];
```

Comment:
Here only using Email field for Current User. You can utilize this logic using UserInfo.getUserEmail.
Please pull latest changes from git and raise PR again with the changes